mdp/hinet/flownode.py

```diff
@@ -142,7 +144,7 @@ def get_train_function(_i_node, _node):
             # the nodes in front of the current nodes.
             # using nested scopes here instead of default args, see pep-0227
```

Comment:
I think we can delete this line, I think I wrote it when nested scope in Python was something new and fancy ;-) (or at least seemed like that to me...)